addons/l10n_jo_edi_extended/models/account_move.py

```diff
@@ -0,0 +1,80 @@
+from odoo import _, api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_jo_edi_invoice_type = fields.Selection(
+        selection=[
+            ('local', 'Local'),
+            ('export', 'Export'),
+            ('development', 'Development Area'),
+        ],
+        string="Invoice Type",
+        help="Invoice Types as per the Income and Sales Tax Department for JoFotara",
+        precompute=True,
+        compute='_compute_l10n_jo_edi_invoice_type',
+        readonly=False,
+        store=True,
+        tracking=True,
+    )
+    l10n_jo_edi_state = fields.Selection(selection_add=[('demo', 'Sent (Demo)')])
```

Comment:
Why?  Also, you could already add it in l10n_jo in principle, but people should not mix demo invoices and production ones normally.

Reply:
Indeed, better here, but in no other l10n do we mix both like that.  It is confusing because you have entries in your accounting that might have been just for test, but they will stay in your accounting...